### PR TITLE
Change used image to registry.k8s.io

### PR DESCRIPTION
**Pull request by wzshiming**
/kind feature

### CODE REVIEW

File: .github/workflows/releases.yaml [hunk 1]
```diff
@@ -28,7 +28,17 @@ jobs:
...
+        if: github.repository == 'kubernetes-sigs/kwok'
+        env:
+          GH_RELEASE: ${{ github.repository }}
+          IMAGE_PREFIX: registry.k8s.io/kwok
+          BINARY_PREFIX: https://github.com/${{ github.repository }}/releases/download
+        run: |
+          REF=${{ github.ref }}
+          GIT_TAG="${REF##*/}" make cross-build
+      - name: Build binary for ghcr.io
+        if: github.repository != 'kubernetes-sigs/kwok'
```
Comment (wzshiming) [inline .github/workflows/releases.yaml:41]: Retain the use of the ghcr.io image for building forked repo

Comment (Huang-Wei) [reply .github/workflows/releases.yaml:41]: This will only be triggered on the fork repo when a release is created, right?

Comment (wzshiming) [reply .github/workflows/releases.yaml:41]: Yes

**Review by yibozhuang — APPROVED** [review]
/lgtm

**Pull request merged**